test/unit/mock/profiles/myprofile1/inspec.yml

```diff
@@ -0,0 +1,8 @@
+name: myprofile1
```

Comment:
I don't think you need this profile. The wrapper-override is vendored.

Reply:
Yup, good catch!